### PR TITLE
Add East Carolina University

### PR DESCRIPTION
**Pull request by cardassian-tailor**
Hope i did this properly

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -117,6 +117,17 @@ websites:
...
       tfa: No
+      
+    - name: East Carolina University
+      url: https://www.ecu.edu
+      img: 3232eculogo.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      twitter: EastCarolina
+      facebook: EastCarolina
```
Comment (mxxcon) [inline _data/education.yml]: No need for `twitter` and `facebook` if `tfa:yes`

**Comment by cardassian-tailor** [comment]
@mxxcon does this look right?

**Review by mxxcon — CHANGES_REQUESTED** [review]
However you saved that logo file it grew in size 😅 My image is 1.5kb, your is 3.2kb

File: _data/education.yml (hunk 1 above)
Comment (cardassian-tailor) [reply _data/education.yml]: ha yeah i remade the logo in photoshop. let me try and shrink it

Comment (mxxcon) [reply _data/education.yml]: Also please add `software: Yes` since your docs link also mentions they support software-based method.

Comment (cardassian-tailor) [reply _data/education.yml]: tfa = two factor auth right? I'm used to seeing this as 2fa.

**Comment by cardassian-tailor** [comment]
@mxxcon There we go how is this

**Comment by mxxcon** [comment]
It's still 3.5kb 😅  https://github.com/cardassian-tailor/twofactorauth/blob/1b9aa12316fed0e33cb36128b449dbcc499c9b49/img/education/3232eculogo.png

Use this image https://user-images.githubusercontent.com/847140/34808014-f31a75ac-f659-11e7-8fbe-59bbf55df630.png

**Comment by Carlgo11** [comment]
I updated the logo and changed the doc link to https as the site supports that. Thanks for your pull request @cardassian-tailor! 😃 

Tavis is having a bad day today due to some earlier GitHub outages and so atm the build fails.
I think it's just to restart the build in a few hours.

**Review by mxxcon — APPROVED** [review]

**Review by jamcat22 — APPROVED** [review]
Thanks for the PR @cardassian-tailor!

**Pull request merged**